feature-buy-impl/src/main/java/io/novafoundation/nova/feature_buy_impl/domain/providers/BanxaProvider.kt

```diff
@@ -19,9 +20,27 @@ class BanxaProvider(
 
     override val name: String = "Banxa"
     override val officialUrl: String = "banxa.com"
-    override val icon: Int = R.drawable.ic_banxa
+    override val logoRes: Int = R.drawable.ic_banxa_provider_logo
+    override val descriptionRes: Int = R.string.banxa_provider_description
 
-    override fun createIntegrator(chainAsset: Chain.Asset, address: String): ExternalProvider.Integrator {
+    override val supportedFlows = setOf(TradeTokenRegistry.TradeFlow.BUY)
+
+    override fun getPaymentMethods(tradeFlow: TradeTokenRegistry.TradeFlow): List<TradeTokenRegistry.PaymentMethod> {
+        return when (tradeFlow) {
+            TradeTokenRegistry.TradeFlow.BUY -> listOf(
+                TradeTokenRegistry.PaymentMethod.Visa,
+                TradeTokenRegistry.PaymentMethod.MasterCard,
+                TradeTokenRegistry.PaymentMethod.ApplePay,
+                TradeTokenRegistry.PaymentMethod.GooglePay,
+                TradeTokenRegistry.PaymentMethod.Sepa,
+                TradeTokenRegistry.PaymentMethod.Other(5)
+            )
+
+            TradeTokenRegistry.TradeFlow.SELL -> throw IllegalStateException("Sell is not supported for Banxa provider")
+        }
+    }
```

Comment:
Minor: its not ideal that we have to handle "sell is not supported" twice. In particular, that we allow to invoke this class with `SELL` even when `SELL` is not supported
This can be solved, for example, by exposing only`getPaymentMethods` and returning empty list for `SELL`